src/tests/function/div_mod.rs

```diff
@@ -0,0 +1,166 @@
+use crate::*;
+
+test_case!(div_mod, async move {
+    use Value::{Null, F64, I64};
+    let eval_div = |dividend: f64, divisor: f64| {
+        let result = dividend / divisor;
+        result as i64
+    };
+    let eval_mod = |dividend: f64, divisor: f64| {
+        let result = (dividend % divisor) as f32;
+        result as f64
+    };
```

Comment:
Thanks for the review. Good to know more about closures in Rust 😄.

The reason why I implemented `f32`-`f64` in modulo is because Rust somehow fails to calculate modulo between two `f64` typed data when its return type is `f64`.

The following output is from test when modulo is performed **without** `f32`-`f64` conversion
```console
# MOD(12.34, 56.78) 
expected: [I64(0), F64(12.34)]
   found: [I64(0), F64(12.34000015258789)]', src/tests/tester.rs:82:17
``` 

Plz let me know if there's a better solution for this, cause for me also that conversion is pretty unnatural 😓

Reply:
This is caused by precision limit of floating point number representation.

That's also why that you add `eval_div` and `eval_mod` rather than using calculated values.
`MOD(12.34, 56.78)` result must be same as `12.34 % 56.78`

That test fails because of this
![image](https://user-images.githubusercontent.com/2025065/130182556-408dbbb8-d4d9-4b6b-be1b-65604667c5ff.png)
You don't need to `as f32` in here.

Reply:
Okay. I'll change it to original `%`.